packages/odyssey-react-mui/src/TagList.tsx

```diff
@@ -0,0 +1,40 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Stack, Tag } from ".";
+import { memo, ReactElement, useMemo } from "react";
+import { ChipComponentType, TagListContext } from "./TagListContext";
+
+export type TagListProps = {
+  children: ReactElement<typeof Tag> | Array<ReactElement<typeof Tag>>;
+};
+
+const TagList = ({ children }: TagListProps) => {
+  const providerValue = useMemo(
+    () => ({
+      chipComponent: "li" as ChipComponentType,
+    }),
+    []
+  );
```

Comment:
Pretty sure we can type this object to avoid `as`:
```suggestion
const TagList = ({ children }: TagListProps) => {
  const providerValue = useMemo(
    () => ({
      chipComponent: "li",
    }: {
      chipComponent: ChipComponentType
    }),
    []
  );
```

Reply:
I would also rename `chipComponent` to `chipElementType` or something referencing the fact that it's not a React component object.

I would assume I could do `<div>{chipComponent}</div>`, and I can, but it doesn't render a chip, it renders the text `'li'` or `'div'`.